app/config/services.yml

```diff
@@ -20,7 +20,7 @@ services:
         resource: '../../src/AppBundle/*'
         # you can exclude directories or files
         # but if a service is unused, it's removed anyway
-        exclude: '../../src/AppBundle/{Entity,Repository,Tests}'
+        exclude: '../../src/AppBundle/{Entity,Tests}'
```

Comment:
concerve Repository for exclude repository not use on service

Reply:
Même si les repository sont des ServiceEntityRepository ?